prove/Develop05/Program.cs

```diff
@@ -22,41 +22,47 @@ static void Main(string[] args)
 
             if (userInput == 1)
             {
-                trackGoal.DisplayGoalTypes();
-                int prompt = int.Parse(Console.ReadLine());
+                int prompt;
 
-                switch (prompt)
+                do
                 {
-                    case 1:
-                        simpleGoal.DisplayStartMessage();
-                        trackGoal.AddGoal(simpleGoal);
-                        break;
-                    case 2:
-                        eternalGoal.DisplayStartMessage();
-                        trackGoal.AddGoal(eternalGoal);
-                        break;
-                    case 3:
-                        checklistGoal.DisplayStartMessage();
-                        trackGoal.AddGoal(checklistGoal);
-                        break;
-                    default:
-                        progressionGoal.DisplayStartMessage();
-                        trackGoal.AddGoal(progressionGoal);
-                        break;
-                }
+                    trackGoal.DisplayGoalTypes();
```

Comment:
This structure doesn't allow you to exit the part where you are adding goals.

Reply:
Fixed